internal/common/fs.go

```diff
@@ -0,0 +1,35 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package common
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+func FindRepositoryRootDirectory() (string, error) {
```

Comment:
What was wrong with the `builder` directory?

Reply:
I wanted to use this function in `docs` package to be able to get the default location of the yaml file with links when generating the readme.
As I thought this function was more related to files than the builder process (looking for the root of the working copy), I was thinking to move this function to another package.

As you mentioned, `common` is not the best place, what about moving it to `files` packages (e.g. internal/files/scm.go)? if it is not clear, I would revert this change and keep it under `builder` package.

Reply:
The `internal/files` directory seems to be more precise location. Alternatively, you can consider `internal/git/repository`. It doesn't look like we will support other SCMs.